### PR TITLE
Add successful skip for API failures

### PR DESCRIPTION
**Pull request by colmsnowplow**

### CODE REVIEW

File: 3-enrich/scala-common-enrich/src/test/resources/iglu-schemas/com.snowplowanalytics.snowplow.enrichments/api_request_enrichment_config/jsonschema/1-0-0 [hunk 1]
```diff
@@ -0,0 +1,167 @@
+{
```
Comment (chuwy) [inline 3-enrich/scala-common-enrich/src/test/resources/iglu-schemas/com.snowplowanalytics.snowplow.enrichments/api_request_enrichment_config/jsonschema/1-0-0]: If you need thse schemas temporarily for tests - put them into a separate commit marked with "temp".

File: 3-enrich/scala-common-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/enrichments/registry/apirequest/ApiRequestEnrichmentIntegrationTest.scala [hunk 2]
```diff
@@ -309,4 +474,41 @@ class ApiRequestEnrichmentIntegrationTest extends Specification with ValidationM
...
+               List(IntegrationTests.customContexts),
+               List(IntegrationTests.unstructEvent)))
+
+    val context2 = config2.flatMap(
+      _.lookup(event,
+               List(IntegrationTests.weatherContext),
+               List(IntegrationTests.customContexts),
+               List(IntegrationTests.unstructEvent)))
+
+    println(context2)
```
Comment (chuwy) [inline 3-enrich/scala-common-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/enrichments/registry/apirequest/ApiRequestEnrichmentIntegrationTest.scala]: Shouldn't be there

File: 3-enrich/scala-common-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/enrichments/registry/apirequest/ApiRequestEnrichmentIntegrationTest.scala [hunk 3]
```diff
@@ -309,4 +474,41 @@ class ApiRequestEnrichmentIntegrationTest extends Specification with ValidationM
       }
     }
   }
+
+  def e3 = {
+    val config  = ApiRequestEnrichmentConfig.parse(IntegrationTests.configuration3, SCHEMA_KEY)
```
Comment (chuwy) [inline 3-enrich/scala-common-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/enrichments/registry/apirequest/ApiRequestEnrichmentIntegrationTest.scala]: I think we just should put the `configuration` into this spec, rather than referring to them as to `1`/`2`/`3`

**Comment by benjben** [comment]
Closing as Anton's review didn't get addressed and enrich got moved to its new home.
